docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node:v0.22.0
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
+      postgres_db: graph-node
+      ipfs: 'ipfs:5001'
+      # Change next line if you want to connect to a different JSON-RPC endpoint
+      ethereum: 'mainnet:http://host.docker.internal:8545'
+      RUST_LOG: info # up until v0.22.0
```

Comment:
Lets just remove `RUST_LOG` and keep only `GRAPH_LOG`.

Reply:
Done. Please approve 🐶